diplo.lua

```diff
@@ -0,0 +1,88 @@
+--[[
+diplo - Quick "Diplomacy"

```

Comment:
@lethosor is our resident naming expert. I'll let him weigh in on whether this is an appropriate name.

Reply:
I couldn't decide between 'diplomat', 'diplomacy' or the like, so i just opted for something short. Name doesn't really matter as long as it stays at a convenionet length to type, imo.

Reply:
even length doesn't matter as much as it used to now that `gui/launcher` has autocomplete.

Reply:
Sorry for the delay - `diplomacy` is my suggestion from a quick read-through of the script.